examples/CMakeLists.txt

```diff
@@ -17,6 +17,13 @@ set(ASYNC_SIMPLE_SOURCE_FILES
 add_executable(async_simple ${ASYNC_SIMPLE_SOURCE_FILES})
 target_link_libraries(async_simple PRIVATE lifthttp -Wl,-\( ${LIBSSL} ${LIBCRYPTO} ${LIBCURL} ${LIBCARES} -Wl,-\) uv pthread z dl)
 
+### async_simple_max_bytes ###
+set(ASYNC_SIMPLE_SOURCE_FILES
```

Comment:
this is resetting the ASYNC_SIMPLE_SOURCE_FILES variable, shouldn't it be ASYNC_SIMPLE_MAX_BYTES_SOURCE_FILES?

Reply:
Yes, it should! Fixed!